Server/Server.py

```diff
@@ -29,10 +31,9 @@ def __init__(self):
             self.__serverCommunicatorsList = []
 
     def receive_message(self, message, ID):
-        print(f"Client {ID} sent:", message)
-        message_split = message.split(";")
-        if message_split[0] == "csoki":
-            self.send_message("koszonom", ID)
+        if message["type"] == server_message_constants.MessageType.CON:
```

Comment:
As far as I remember, client never sends server message constant. It is NOT good if we have a duplicate of every contstant and both client, server sends, and checks their own copy.